pages/loading.tsx

```diff
@@ -128,20 +131,28 @@ export const LoadingPage: NextPage = () => {
     [processData.result]
   );
 
+  const router = useRouter();
+  const precacheProcessPages = usePrecacheAll(router);
```

Comment:
Suggest rolling `useRouter` into `usePrecacheAll`, since we don't do anything with it in this scope.